src/js/navIcons.js

```diff
@@ -47,12 +47,15 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             that.locate("confirmedIndicator").addClass(that.options.styles.show);
         }
     };
-
+    
```

Comment:
Removed, but mysteriously this is not detected by our current linting configuration